<launch>
<rosparam param="Section0_AutoFill">
[[0.48170500280236367, 0.7512243292280542, -0.1691877719992555, -1.0943618695469643, 2.8691538151571225, -0.1001539128228116],
[0.5059462620396442, 1.1740508516844046, -0.12768207865832595, -0.7819869012582111, 2.955649606923243, -0.18682464580473446],
[-0.36402360484087076, 1.1998764058862161, -0.1508961633319236, 0.2829778716865471, 3.0892745998450306, -0.24017379613643758],
[-0.37124690934772825, 0.818824753926981, -0.14593427107081242, 0.4139132599780484, 3.08629307289802, -0.16402398369383053]]
</rosparam>

<rosparam param="Section1_AutoFill">
[[0.7512243292280542, 0.48170500280236367, -0.1691877719992555, -1.0943618695469643, 2.8691538151571225, -0.1001539128228116],
[1.1740508516844046, 0.5059462620396442, -0.12768207865832595, -0.7819869012582111, 2.955649606923243, -0.18682464580473446],
[1.1998764058862161, -0.36402360484087076, -0.1508961633319236, 0.2829778716865471, 3.0892745998450306, -0.24017379613643758],
[0.818824753926981, -0.37124690934772825, -0.14593427107081242, 0.4139132599780484, 3.08629307289802, -0.16402398369383053]]
</rosparam>


<rosparam param="Section2_AutoFill">
[[0.44644750841084624, -0.6865354065837639, -0.09392428159159258, -2.43962962332685, 1.666061689869301, -0.27562096678885867],
[0.49030723375081003, -1.105811758069206, -0.09626158706498708, -2.4725530338191284, 1.772836970233566, -0.40889364213911494],
[-0.2387369271316873, -1.1820826089949035, -0.09345090310925779, -1.83708064401468, 2.4439407600916803, -0.15784159801877626],
[-0.2665619835493377, -0.7704546280045004, -0.09407002972475523, -1.5844716130427738, 2.6120578188680574, -0.05688860952155684]


]

</rosparam>

<rosparam param="Section3_AutoFill"> 
[[-0.7512243292280542, 0.48170500280236367, -0.1691877719992555, -1.0943618695469643, 2.8691538151571225, -0.1001539128228116],
[-1.1740508516844046, 0.5059462620396442, -0.12768207865832595, -0.7819869012582111, 2.955649606923243, -0.18682464580473446],
[-1.1998764058862161, -0.36402360484087076, -0.1508961633319236, 0.2829778716865471, 3.0892745998450306, -0.24017379613643758],
[-0.818824753926981, -0.37124690934772825, -0.14593427107081242, 0.4139132599780484, 3.08629307289802, -0.16402398369383053]]
</rosparam>

<rosparam param="NodDifferentials"> 
[[0.037507658513412666, -0.34971099261284644, -0.5078117290432184, 1.092004162634252, -1.4106272197127374, 1.5709524971042654],
[0.01726614251195916, -0.38925011958357536, -0.3449850643193274, 1.8581107756495059, -1.8822105309956962, 0.9514782625043029]]
</rosparam>

<rosparam param="WithdrawPose"> 
[0.10289531452998273, -0.3537502847578775, -0.6997784285870847, 1.5036295459032858, -1.7234581095793673, 1.0822197379665612]
</rosparam>

<rosparam param="leftRetreat"> 
[-0.4674027848787509, -0.47344564484658175, -0.3020315736283644, 2.846451382018769, -1.2879693442564653, 0.10305285229426728]
</rosparam>

<rosparam param="RightRetreat"> 
[0.5535976209158624, -0.48982789775225505, -0.26622883152663945, 0.9360179387789933, -2.9862472469635284, -0.011117096130778686]
</rosparam>

<param name="AutoCalibrate" type="bool" value="True" />
<param name="AnimationsFileLoc" type="str" value="/home/naodev/Documents/default_ROSws/src/ur10DrawingSocial/src/AnimationCMD.csv" />
<param name="ImagesPath" type="str" value="/home/naodev/Documents/default_ROSws/src/ur10DrawingSocial/robot_img_v2/" />
<param name="speedParameters" type="double" value="0.3" />
<param name="roboIP" type="str" value="10.0.0.157" />

<node name="robo_draw" pkg="robo_draw" type="robo_draw" output="screen" launch-prefix="gnome-terminal --command"/>

<node name="interfaceNode" pkg="ur10DrawingSocial" type="roboInterface.py" output="screen" launch-prefix="gnome-terminal --command"/>

<node name="drawingCmd" pkg="ur10DrawingSocial" type="DrawingCmd.py" output="screen" launch-prefix="gnome-terminal --command"/>


</launch>
